gradle-baseline-java-config/resources/checkstyle/checkstyle.xml

```diff
@@ -370,6 +370,11 @@
             <property name="format" value="(void setUp\(\))|(void setup\(\))|(void setupStatic\(\))|(void setUpStatic\(\))|(void beforeTest\(\))|(void teardown\(\))|(void tearDown\(\))|(void beforeStatic\(\))|(void afterStatic\(\))"/>
             <property name="message" value="Test setup/teardown methods are called before(), beforeClass(), after(), afterClass(), but not setUp, teardown, etc."/>
         </module>
+        <module name="RegexpSinglelineJava">
```

Comment:
we should make sure it's still possible to call this in test code (e.g. DFE is not enforced on test code), since JUnit provides no easy way to run something after all tests

Reply:
Is @SuppressWarnings("BanShutdownHooks") not enough? I think I can also add it to the list of suppressions.

Reply:
Done